src/python/WMCore/WMSpec/Steps/Executor.py

```diff
@@ -19,6 +19,14 @@
 getStepName = lambda step: WMStepHelper(step).name()
 getStepErrorDestination = lambda step: WMStepHelper(step).getErrorDestinationStep()
 
+#TODO this function can me moved to Utils.
+def which(name):
+    for path in os.getenv("PATH").split(os.path.pathsep):
+        full_path = path + os.sep + name
+        if os.path.exists(full_path):
+            return full_path
+    return None
+
```

Comment:
This code doesn't check whether file is executable (assumes condor_chirp is executable).